config/routes.rb

```diff
@@ -25,6 +25,7 @@
       post 'audit'
     end
   end
+  get 'files/new/in_collection/:collection_id' => 'generic_files#new', as: 'generic_files_upload_in_collection'
```

Comment:
I'm probably overlooking something obvious, but why do we need a dedicated route for this rather than using the existing `/files/new` route with a param passed to it (e.g., `/files/new?collection=abc123def`)?